README.md

```diff
@@ -15,11 +15,10 @@ Extensions make Piwigo easily customizable. Icing on the cake, Piwigo is free an
 
 ## Screenshots
 
-![](http://piwigo.org/screenshots/homepage/piwigo-batch-manager.png)
+![](https://camo.githubusercontent.com/e6a5d3a1200eabf27760b660a9b0505418ac14b9/68747470733a2f2f70697769676f2e6f72672f73637265656e73686f74732f6769746875622d73637265656e73686f742d322e31302e6a7067)

```

Comment:
Eeeeeh, granted that may work but are you sure this link will keep existing ? I'm 80% sure that this is probably a tmp/cache location that github uses in its internal to optimize bandwith etc but shouldnt be used as the main source for an image as it may get deleted anytime ?